tools/ci/actions/runner.sh

```diff
@@ -70,8 +70,13 @@ Build () {
 }
 
 Test () {
-  echo Running the testsuite
-  $MAKE -C testsuite parallel
+  if [ "$1" = "sequential" ]; then
+    echo Running the testsuite sequentially
+    $MAKE -C testsuite all
+  else
+    echo Running the testsuite
+    $MAKE -C testsuite parallel
+  fi
```

Comment:
Nitpick: I would prefer to require an argument in both case, `sequential` or `parallel` (or an error), instead of defaulting to `parallel` as you are doing, which makes sense given the previous behavior but not really in the result.

Reply:
Done.